lib/lib-shared-models/src/main/java/org/sagebionetworks/repo/model/ServiceConstants.java

```diff
@@ -1,5 +1,7 @@
 package org.sagebionetworks.repo.model;
 
+import org.sagebionetworks.repo.model.favorite.SortBy;
+
```

Comment:
Why is this import added?

Reply:
I was trying to create a string constant using the enum and forgot to remove the import after I realized that I couldn't use it.